app/finders/user/filtered.rb

```diff
@@ -0,0 +1,25 @@
+class User
+  class Filtered < ApplicationFinder
+    def initialize(params = {})
+      @params = params
+      @speaker =  @params.delete(:speaker)
```

Comment:
Can we distinguish `speaker` and checked checkboxes params so that we don't have to remove a key here?